agent/functional_tests/testdata/taskdefinitions/task-server-endpoint-validator/task-definition.json

```diff
@@ -0,0 +1,10 @@
+{
+    "family": "task-server-endpoint-validator",
+    "taskRoleArn": "$$$TASK_ROLE$$$",
+    "networkMode": "host",
+    "containerDefinitions": [{
+        "image": "127.0.0.1:51670/amazon/task-server-endpoint-validator:latest",
+        "name": "container_1",
```

Comment:
nit: rename this to a more descripritive one.

Reply:
do you have a suggestion? i thought the task definition and image name is descriptive enough and i didn't want to introduce similar/same name to cause confusion as it is the only container in the test.

Reply:
Ah what about something like task_server_endpoint_validator_container? 🤔